addons/project_issue/migrations/9.0.1.0/pre-migration.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenUpgrade module for Odoo
+#    @copyright 2014-Today: Odoo Community Association, Microcom
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openupgradelib import openupgrade
+
+
+# Column is kept even though it was removed in v9, as to allow recovery by a future OCA module
+column_renames = {
+    'project.issue': [
+        ('categ_ids', 'tag_ids'),
```

Comment:
@pedrobaeza Can't we just rename the table project_category_project_issue_rel to project_issue_project)tags_rel and its column names and update the res_id names?